DevCycle/Models/Cache.swift

```diff
@@ -64,6 +70,36 @@ class CacheService: CacheServiceProtocol {
         self.remove(key: CacheKeys.anonUserId)
     }
     
+    func getConfigUserId(user: DVCUser) -> String? {
+        let key = getConfigKeyPrefix(user: user)
+        return self.getString(key: "\(key).USER_ID")
+    }
+    
+    func getConfigFetchDate(user: DVCUser) -> Int? {
+        let key = getConfigKeyPrefix(user: user)
+        return self.getInt(key: "\(key).FETCH_DATE")
+    }
```

Comment:
these could be private methods, or this logic could be moved into `getConfig`

Reply:
I'll make these private right now but may move them into `getConfig` in the next PR